src/views/Admin/AdminNetworkingKeys.js

```diff
@@ -330,7 +330,8 @@ export default function AdminNetworkingKeys() {
         {completed && (
           <>
             <Grid.Item full as={NoticeBox} className="mb3">
-              You need this keyfile to authenticate with Arvo.
+              You need this keyfile to authenticate with Arvo. If you have
+              already booted Arvo, reboot with this new keyfile.
```

Comment:
This is only true in the `discontinuous` case.

To use new keys while remaining continuous, you use the `|rekey` command, pasting in the contents of your keyfile as a string. ie `|rekey '0w123.whatever`

That has its own slew of issues (it's kinda strict about when/how you can change keys), but certainly better than telling people to reboot their ships.

Proposed copy: "If you have already booted Arvo, run `|rekey '0wkeyfile.contents'`

I'd include a link to the docs, but that doesn't include this at all. cc @rmariani @baudtack

Reply:
I'm going to update to that copy but this needs more thought around the whole set of flows and documentation etc to be really handled.